routes/reps.js

```diff
@@ -3,6 +3,12 @@
 // const router = express.Router();
 // const repsCtrl = require('../controllers/reps');
 
+<<<<<<< HEAD
+
+
+// GET and POST will go here
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
router.get("/", repsCtrl.index); 
```